File: angular/src/app/services/Lesson/dtos/CreateLessonInput.ts
import { UserPartOutPut } from './../../User/dtos/UserPartOutPut';
export class CreateLessonInput implements ICreateLessonInput{
    name: string;
    teachers: UserPartOutPut[];

    constructor(data?: ICreateLessonInput){}
}

export interface ICreateLessonInput{
    name:string|undefined;
    teachers:UserPartOutPut[]|undefined;
}

File: aspnet-core/src/Lesson.Application/Domain/Lesson/Dto/CreateLessonInput.cs
﻿using Lesson.Authorization.Users;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lesson.Domain.Lesson.Dto
{
    public class CreateLessonInput
    {
        public string Name { get; set; }
        public List<UserPartOutPut> Teachers { get; set; }
    }
}
